apps/dashboard/src/modules/user/components/UserInfo.vue

```diff
@@ -5,6 +5,7 @@
           {{ isGewisUser(props.user) ? props.user.gewisId : `E${props.user.id}` }} <br />
           {{ props.user.firstName }} {{ props.user.lastName }}
         </h1>
+        <p class="font-bold text-red-500 text-center">{{ $t("quickOverview.underAge") }}</p>
```

Comment:
Should there not be conditional rendering on this? Or am I missing something?

Reply:
Hahahahaha this was too hot of a fix, will add condition

Reply:
Should be fixed now